src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -429,11 +429,11 @@
         Parameters may be supplied from a file using the `@{path}` syntax, a JSON string, or as <KEY=VALUE> pairs. Parameters are evaluated in order, so when a value is assigned twice, the latter value will be used.
         It is recommended that you supply your parameters file first, and then override selectively using KEY=VALUE syntax.
   - name: --template-file -f
-    short-summary: The path to the template file or Bicep file.
+    short-summary: The path to the template file or Bicep file. Choose only one of --template-file FILE | --template-uri URI | --template-spec ID to specify.
   - name: --template-uri -u
-    short-summary: The URI to the template file.
+    short-summary: The URI to the template file. Choose only one of --template-file FILE | --template-uri URI | --template-spec ID to specify.
   - name: --template-spec -s
-    short-summary: The template spec resource id.
+    short-summary: The template spec resource id. Choose only one of --template-file FILE | --template-uri URI | --template-spec ID to specify.
```

Comment:
Please add the help information at the command level instead of adding at the parameter level to avoid redundant information

Reply:
Please use a more user-friendly description, such as: `Please specify only one of --template-file FILE | --template-uri URI | --template-spec ID to pass in the template data`

Reply:
Please use a more user-friendly description, such as: `Please specify only one of --template-file FILE | --template-uri URI | --template-spec ID to pass in the template data`